src/pages/index.js

```diff
@@ -335,11 +362,133 @@ const IndexPage = (props) => {
+                                width={isDesktop ? 100 : isMobile ? 50 : 90}
+                                height={isDesktop ? 100 : isMobile ? 50 : 90}
+                                style={{
+                                    marginTop: 'auto',
+                                    marginBottom: 'auto',
+                                }}
+                            />
+                        </Box>
+                        <Box
+                            sx={{
+                                fontSize: isMobile ? 'small' : 'medium',
+                            }}
+                        >
+                            Thank you{' '}
+                            {thankYou?.filter((item) => item?.trim() === '')
+                                .length === 0 && (
+                                <a
+                                    target='_blank'
+                                    href={thankYou[5]?.replace(/['"]+/g, '')}
+                                >
+                                    {thankYou[3]?.replace(/['"]+/g, '')
+                                        ? thankYou[3]?.replace(/['"]+/g, '')
+                                        : thankYou[2]?.replace(/['"]+/g, '')}
+                                </a>
+                            )}
+                            <br />
+                            for making {thankYou[7]?.replace(
+                                /['"]+/g,
+                                ''
+                            )} pull{' '}
+                            {parseInt(thankYou[7]?.replace(/['"]+/g, '')) >= 2
+                                ? 'requests'
+                                : 'request'}
+                            <br />
+                            to{' '}
+                            {thankYou[8]?.split(' ')?.length >= 4
+                                ? parseInt(thankYou[8]?.split(' ')?.length) +
+                                  ' Jenkins'
+                                : 'the'}
+                            {thankYou[8]?.split(' ').length < 4 &&
+                                thankYou[8]
+                                    ?.replace(/['"]+/g, '')
+                                    .split(/\s+/)
+                                    .filter(Boolean)
+                                    .map((repo, idx) => (
+                                        <>
+                                            {2 <=
+                                                thankYou[8]?.split(' ')
+                                                    .length &&
+                                                idx ===
+                                                    thankYou[8]?.split(' ')
+                                                        .length -
+                                                        2 &&
+                                                'and '}
+                                            <a
+                                                target='_blank'
```

Comment:
```suggestion
```
Same as above.

Reply:
That is for opening the link in a new tab, I would suggest keeping it.

Reply:
It is a practice in the industry to use `target='_blank'` for external links like this.

Reply:
Add the `rel="noreferrer"` as suggested in the article cited by @lemeurherve on Gitter https://www.searchenginejournal.com/blank-link-attribute/435883/#:~:text=There%27s%20A%20Security%20Reason%20For,same%20process%20as%20your%20page as a security measure.